lua/lualine/themes/iceberg.lua

```diff
@@ -1,3 +1,43 @@
+if vim.o.background == "light" then
+    return {
+        normal = {
+            a = { bg = "#757ca3", fg = "#e8e9ec" },
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        insert = {
+            a = { bg = "#2d539e", fg = "#e8e9ec" },
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        visual = {
+            a = { bg = "#668e3d", fg = "#e8e9ec" },
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        replace = {
+            a = { bg = "#c57339", fg = "#e8e9ec" },
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        command = {
+            a = { bg = "#a093c7", fg = "#e8e9ec" },
+            b = { bg = "#9fa6c0", fg = "#e8e9ec" },
+            c = { bg = "#cad0de", fg = "#8b98b6" }
+        },
+        terminal = {
+            a = { bg = "#89b8c2", fg = "#e8e9ec" },
```

Comment:
Use cyan from light theme
```suggestion
            a = { bg = "#3f83a6", fg = "#e8e9ec" },
```